sorc/grid_tools.fd/regional_esg_grid.fd/pmat4.f90

```diff
@@ -1769,17 +1938,15 @@ subroutine ctoz(v, z,infz)!                                             [ctoz]
 z=z*zzpi
 end subroutine ctoz
    
-!=============================================================================
+!> ???
+!! @author R. J. Purser
```

Comment:
Missing params.

Reply:
The function with ??? is the one without original documentation as you requested. That is why I left empty on param field. Any suggestion?

Reply:
Yes, list each param, in or out or inout, and put ??? on the description.